src/sparseml/yolov8/trainers.py

```diff
@@ -636,9 +637,7 @@ def export(self, **kwargs):
                 "Recipe checkpoint detected, saving the "
                 f"recipe to the deployment directory {deployment_folder}"
             )
-            ScheduledModifierManager.from_yaml(recipe).save(
-                os.path.join(deployment_folder, "recipe.yaml")
-            )
+            recipe.save(os.path.join(deployment_folder, "recipe.yaml"))
```

Comment:
👀

Reply:
Oh yeah meant to tag you, I was getting issues when using this with one-shot, because one shot creates a recipe. Assume this is needed when one-shot is not present?

Reply:
Updated